src/mpid/ch4/shm/posix/eager/iqueue/iqueue_send.h

```diff
@@ -80,7 +80,7 @@ MPIDI_POSIX_eager_send(int grank, MPIDI_POSIX_am_header_t * msg_hdr, const void
      * tail. */
     cell->payload_size = 0;
     if (am_hdr) {
-        MPI_Aint resized_am_hdr_sz = MPIDI_POSIX_RESIZE_TO_MAX_ALIGN(am_hdr_sz);
+        MPI_Aint resized_am_hdr_sz = MPL_ROUNDUP(am_hdr_sz);
```

Comment:
What I meant is simply use this:
```
MPI_Aint resized_am_hdr_sz = MPL_ROUND_UP_ALIGN(am_hdr_sz, MAX_ALIGNMENT);
```

No need to add additional macro in `mpl_base.h`.

Reply:
Could you also squash into a single commit? Thanks.

Reply:
I understand, and I made the change but the commits are not showing here, but I can see them on the branch for this PR. Sorry for the confusion, working on a fix

Reply:
I see. Are you familiar with git interactive rebasing? https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d

Reply:
Yep, done